src/Agda2Hs/Pragma.hs

```diff
@@ -74,12 +81,13 @@ processPragma :: QName -> C ParsedPragma
 processPragma qn = liftTCM (getUniqueCompilerPragma pragmaName qn) >>= \case
   Nothing -> return NoPragma
   Just (CompilerPragma _ s)
-    | "class" `isPrefixOf` s    -> return $ ClassPragma (words $ drop 5 s)
-    | s == "existing-class"     -> return ExistingClassPragma
-    | s == "unboxed"            -> return $ UnboxPragma Lazy
-    | s == "unboxed-strict"     -> return $ UnboxPragma Strict
-    | s == "transparent"        -> return TransparentPragma
-    | s == newtypePragma        -> return $ NewTypePragma []
-    | "deriving" `isPrefixOf` s -> processDeriving s DefaultPragma
+    | "class" `isPrefixOf` s     -> return $ ClassPragma (words $ drop 5 s)
+    | s == "existing-class"      -> return ExistingClassPragma
+    | s == "unboxed"             -> return $ UnboxPragma Lazy
+    | s == "unboxed-strict"      -> return $ UnboxPragma Strict
+    | s == "transparent"         -> return TransparentPragma
+    | s == newtypePragma         -> return $ NewTypePragma []
+    | "strategy:" `isPrefixOf` s -> return $ DerivePragma (parseStrategy (drop (length "strategy:") s))
```

Comment:
I'm in favor of dropping the `strategy:` (do you even check "strategy" is written?) as the fact you're compiling an instance makes it redundant.

Reply:
I've changed this to be included as part of the `derive` clause.